src/components/LoginModal.css

```
@@ -0,0 +1,41 @@
+
+.login-modal__header h2 {
+  margin: 0;
+  font-size: 18px;
+  font-weight: 800;
+  letter-spacing: -0.02em;
+  color: var(--brand-primary, #3182f6);
+}
+
+.login-modal__header ion-button {
+  --color: var(--text-muted, #8b95a1);
+  margin: 0;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)